types/instafeed.js/package.json

```
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/instafeed.js",
+    "version": "2.0.9999",
+    "projects": [
+        "https://instafeedjs.com"
+    ],
+    "devDependencies": {
+        "@types/instafeed.js": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Luis Sousa",
+            "githubUsername": "DefinitelyTyped"
```

shouldn't this read:
```suggestion
            "githubUsername": "IncognitaDev"
```
(only user accounts can be owners, DT just takes ownerships if types definition goes derelict)

---

I think this has been added by the create command

---

Yeah, typically this means that your local git information was not enough to determine the right value (so it picks a default). The default should probably be the empty array (no owners used to be illegal).